src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -2122,6 +2122,81 @@ public static int indexOfAny(final CharSequence cs, final String searchChars) {
         return indexOfAny(cs, searchChars.toCharArray());
     }
 
+    /**
+     * <p>Search a CharSequence to find the last index of any
```

Comment:
Use the active voice in Javadoc: "Returns...", "Searches...", "Gets..." and so on.